media/js/firefox/australis/fx36/whatsnew.js

```diff
@@ -0,0 +1,53 @@
+;(function($, Mozilla) {
+    'use strict';
+
+    var $tourStrings = $('#strings');
+    var helloAnimationStage = $('#hello-animation-stage');
+
+    //track if this is the first time a user has seen tour
+    function trackFirstTimeUse() {
+        var firstTime = 'True';
+        try {
+            if (localStorage.getItem('mozUITourGlobalFlag') === 'taken') {
+                firstTime = 'False';
+            } else {
+                localStorage.setItem('mozUITourGlobalFlag', 'taken');
+            }
+            gaTrack(['_trackEvent', 'Tour Interaction', 'First Time Seeing Tour', firstTime, 0, true]);
```

Comment:
Actually I do also wonder if this event is still needed or not. It was copy/pasted from the original australis whatsnew tour, so maybe not applicable now? This will be the first (and probably only) time people will see this one regardless, as it differs from firstrun. Might actually make sense to remove it now?